chat/basics/android.md

```diff
@@ -371,7 +376,7 @@ chatContainer.sendMessage(conversation,
                           "Hello!",
                           null,
                           null,
-                          new SaveCallback<Message>() {
+                          new SaveMessageCallback() {
```

Comment:
After we changed back, the callback of save message is `SaveCallback<Message>` or `SaveMessageCallback`?

Reply:
@carmenlau @cheungpat  It should be `SaveCallback<Message>`.